src/CMakeLists.txt

```diff
@@ -57,13 +56,22 @@ install (TARGETS
         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
 )
 
-set(PKGCONFIG_CFLAGS)
-set(PKGCONFIG_REQUIRES
-    base-types
-    base-logging
-    boost_serialization
-    pcl_io-${PCL_VERSION_MAJOR}.${PCL_VERSION_MINOR}
+set(PKG_CONFIG_DEPS
+  base-types
+  base-logging
+  boost_serialization
+  pcl_io-${PCL_VERSION_MAJOR}.${PCL_VERSION_MINOR}
 )
+
+set(pkg_config_dependencies "")
+
+foreach(value ${PKG_CONFIG_DEPS})
+    set(pkg_config_dependencies "${pkg_config_dependencies} ${value}")
+endforeach()
+
+set(PKGCONFIG_REQUIRES ${pkg_config_dependencies})
```

Comment:
This could be done by `list(JOIN PKG_CONFIG_DEPS " " PKGCONFIG_REQUIRES)`

--> https://cmake.org/cmake/help/latest/command/list.html#reading

Reply:
Works as well, thanks!